packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -130,6 +141,11 @@ html.interface-interface-skeleton__html-container {
 	flex-shrink: 0;
 	border-top: $border-width solid $gray-200;
 	color: $gray-900;
+	position: fixed;
+	bottom: 0;
+	width: 100%;
+	background-color: $white;
+	z-index: 90;
```

Comment:
**Note:**

The sidebar border was overlapping the footer, which is why we set an explicit z-index of 90. I'm not certain where this originates from, but I'll be looking more into it.

Reply:
I just found this file: https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_z-index.scss  See line 83-84-85

Might be helpful for you and we could add this to the mapping with a comment explaining the value.

Reply:
Good point @david-szabo97, and it's a good reminder to use the [`z-index`](https://github.com/WordPress/gutenberg/blob/a8b872408e751b78ad9790ddc1f0446c0577e32e/packages/base-styles/_z-index.scss#L143-L149) mixin (and add the key to the list) when defining new z-indices.